scripts/stac/tests/generate_title_test.py

```diff
@@ -4,131 +4,133 @@
     metadata_auck, _ = metadata
-    title = "Auckland 0.3m Rural Aerial Photos (2023)"
+    title = "Auckland 0.3m Forest assessment Rural Aerial Photos (2023)"
     collection = ImageryCollection(metadata_auck)
     assert collection.stac["title"] == title
 
 
-def test_generate_dem_title(metadata: Tuple[CollectionTitleMetadata, CollectionTitleMetadata]) -> None:
+def test_generate_dem_title(metadata: Tuple[CollectionMetadata, CollectionMetadata]) -> None:
     metadata_auck, _ = metadata
-    metadata_auck["category"] = "DEM"
+    metadata_auck["category"] = "dem"
     collection = ImageryCollection(metadata_auck)
-    title = "Auckland LiDAR 0.3m DEM (2023)"
+    title = "Auckland - Forest assessment LiDAR 0.3m DEM (2023)"
     assert collection.stac["title"] == title
 
 
-def test_generate_dsm_title(metadata: Tuple[CollectionTitleMetadata, CollectionTitleMetadata]) -> None:
+def test_generate_dsm_title(metadata: Tuple[CollectionMetadata, CollectionMetadata]) -> None:
     metadata_auck, _ = metadata
-    metadata_auck["category"] = "DSM"
+    metadata_auck["category"] = "dsm"
     collection = ImageryCollection(metadata_auck)
-    title = "Auckland LiDAR 0.3m DSM (2023)"
+    title = "Auckland - Forest assessment LiDAR 0.3m DSM (2023)"
     assert collection.stac["title"] == title
 
 
-def test_generate_satellite_imagery_title(metadata: Tuple[CollectionTitleMetadata, CollectionTitleMetadata]) -> None:
+def test_generate_satellite_imagery_title(metadata: Tuple[CollectionMetadata, CollectionMetadata]) -> None:
     metadata_auck, _ = metadata
-    metadata_auck["category"] = "Satellite Imagery"
+    metadata_auck["category"] = "satellite-imagery"
     collection = ImageryCollection(metadata_auck)
-    title = "Auckland 0.3m Satellite Imagery (2023)"
+    title = "Auckland 0.3m Forest assessment Satellite Imagery (2023)"
     assert collection.stac["title"] == title
 
 
-def test_generate_historic_imagery_title(metadata: Tuple[CollectionTitleMetadata, CollectionTitleMetadata]) -> None:
+def test_generate_historic_imagery_title(metadata: Tuple[CollectionMetadata, CollectionMetadata]) -> None:
     title = "Auckland 0.3m SNC8844 (2023)"
     metadata_auck, _ = metadata
-    metadata_auck["category"] = "Aerial Photos"
+    metadata_auck["category"] = "aerial-photos"
     metadata_auck["historic_survey_number"] = "SNC8844"
     collection = ImageryCollection(metadata_auck)
     assert collection.stac["title"] == title
 
 
-def test_generate_title_long_date(metadata: Tuple[CollectionTitleMetadata, CollectionTitleMetadata]) -> None:
+def test_generate_title_long_date(metadata: Tuple[CollectionMetadata, CollectionMetadata]) -> None:
     metadata_auck, _ = metadata
     metadata_auck["end_datetime"] = datetime(2024, 1, 1)
     collection = ImageryCollection(metadata_auck)
-    title = "Auckland 0.3m Rural Aerial Photos (2023-2024)"
+    title = "Auckland 0.3m Forest assessment Rural Aerial Photos (2023-2024)"
```

Comment:
how come all of these have "Forest assement" it would be way more common not to have this in the title?